packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/lambda-walkthrough.ts

```diff
@@ -100,10 +126,62 @@ export async function updateWalkthrough(context, lambdaToUpdate?: string) {
   const projectBackendDirPath = context.amplify.pathManager.getBackendDirPath();
   const resourceDirPath = path.join(projectBackendDirPath, category, functionParameters.resourceName);
   const currentParameters = loadFunctionParameters(context, resourceDirPath);
+  const functionRuntime = context.amplify.readBreadcrumbs(context, category, functionParameters.resourceName).functionRuntime as string;
```

Comment:
@edwardfoyle Is there a better way to get the function's runtime here?

Reply:
I guess we can only get the runtime from amplify.state file as we arent runtime in parameters file